### ISSUE TITLE
Cannot use 'in' operator to search for 'aria-controls' in null

### ISSUE THREAD

**Issue by Yusufzai** [TarekRaafat/autoComplete.js#311]
I am getting the following error when I am trying to put the same example in autocomplete website

https://prnt.sc/26jzq49

Note: I am using it with Bootstrap 5

**Comment by folknor** [comment]
![index](https://user-images.githubusercontent.com/233791/151376076-78501dd8-c5a7-4631-b5da-f2b8467da387.png)

That probably means your input element isn't found. Update `selector` accordingly.

https://tarekraafat.github.io/autoComplete.js/#/configuration?id=selector-optional

**Comment by ikramooe** [comment]
i'm having the same issue

**Comment by elanclarkson** [comment]
In my case this was happening because I had foolishly copy/pasted `const autoCompleteJS = new autoComplete({ config });` from the docs, but I had already defined `config` as an object so it should actually be `const autoCompleteJS = new autoComplete(config);`.

Hope this helps if anyone else did the same thing!

**Comment by thorsten** [comment]
I have the same issue, I'm also using Bootstrap v5.1

**Comment by marcoschneider** [comment]
I have the same issue, using jquery.

**Comment by marcoschneider** [comment]
For every one still having this issue. This error message probably means that the selector you pass in the configs was not found in the DOM. Just make sure the element is rendered in the DOM before you make the autoComplete instance.

**Comment by escopecz** [comment]
Another option is that the JS is executed before the DOM is ready. Wrap your code with
```js
window.onload = function () {
    
}
```

**Comment by JLidholm** [comment]
I see the solution on defining a custom 'selector' in the config, but in genereal the code seems purposely built to only handle one input with the default id 'autoComplete' Any other configuration (I'm going to need at least two auto-completing inputs on my page) seems to break the list functionality in one way or the other.

Running on Bootstrap 5

Jus tto mention one issue I found - assignment of ctx.name and the ctx selector in function configure (in v10.2.7 in autoComplete.js:199).
Tha variable name is assigned at the top, but then in the options loop, ctx.name can get changed if name is defined in the config, but the following code still uses the initial name variable with the default value of 'autoComplete'.
